src/core/include/config.h

```diff
@@ -107,6 +109,8 @@ class Config {
   struct MapServer {
     std::string channelName = "Athena";
     std::string charIp = "127.0.0.1";
+    std::string charUser = "char";
+    std::string charPassword = "098f6bcd4621d373cade4e832627b4f6";
```

Comment:
We should probably use only one user/password for all servers, otherwise it'll just be impossible if/when we have multiple map servers

Reply:
Why would this be impossible? You can use one user/password for all servers if you wanted to.

Reply:
alright